utils/src/main/java/io/scalecube/utils/CopyingModifier.java

```diff
@@ -5,11 +5,18 @@
 import java.lang.reflect.Constructor;
 import java.util.function.Consumer;
 
+/**
+ * Utility functional object for calling copy-constructor and appying specified lamda on a copy.
+ * 
+ * @param <E> element type on which to apply operation
+ */
```

Comment:
Actually, I just don't like this interface. There are many other ways to do that.